### PR TITLE
Js licenses

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

**Comment by rullzer** [comment]
Next steps?

**Comment by skjnldsv** [comment]
> Next steps?

profit

**Comment by nickvergessen** [comment]
Node build failed

**Review by ChristophWurst — APPROVED** [review]
Looks who's faking their way to the top of the contributor stats

**Review by blizzz — APPROVED** [review]

**Comment by skjnldsv** [comment]
Fixed script, js licenses and copyrights
Waiting for ci

File: apps/files_sharing/src/services/ConfigService.js [hunk 1]
```diff
@@ -11,7 +13,7 @@
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY without even the implied warranty of
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
```
Comment (kesselb) [inline apps/files_sharing/src/services/ConfigService.js:16]: ?

File: apps/files_sharing/src/services/ExternalLinkActions.js [hunk 2]
```diff
@@ -12,7 +12,7 @@
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```
Comment (kesselb) [inline apps/files_sharing/src/services/ExternalLinkActions.js:15]: ?

**Pull request merged**